### PR TITLE
Report Found Item Page

### PR DESCRIPTION
**Pull request by Centerville1**
Report found item page, with buttons linking to it from the home page widget and the main lost and found page.
Screen Shot 2024-11-26 at 17 11 19

### CODE REVIEW

File: src/views/CampusSafety/views/LostAndFound/index.tsx [hunk 1]
```diff
@@ -169,34 +167,47 @@ const LostAndFound = () => {
...
                 <Grid item xs={12} sm={11}>
                   <Typography variant="body1" align="center">
-                    Police staff will view reports, and you will be notified if your item is found.
+                    <InfoOutlined /> Police staff will view reports, and you will be notified if
+                    your item is found.
                   </Typography>
+                  {/* Button to Report a Missing Item */}
+                  <Grid
+                    container
+                    justifyContent="center"
```
Comment (erandersen21) [inline src/views/CampusSafety/views/LostAndFound/index.tsx:178]: The grids don't seem to be properly spaced on the page, so this line doesn't work as intended (at least when I run it). 
Screenshot 2024-11-29 at 10 08 46 PM

File: src/views/CampusSafety/views/LostAndFound/views/ReportFound/index.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  const navigate = useNavigate();
+
+  return (
+    <>
+      <Header />
+      <Card>
+        <CardHeader title={'Report a Found Item'} className="gc360_header" />
+        <CardContent>
+          <Typography>If you've found an item somewhere on campus:</Typography>
+          <ul>
```
Comment (erandersen21) [inline src/views/CampusSafety/views/LostAndFound/views/ReportFound/index.tsx:16]: This is just my opinion, but I think this looks a little weird as a bulleted list.

File: src/views/CampusSafety/views/LostAndFound/views/ReportFound/index.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import { useNavigate } from 'react-router';
+import Header from 'views/CampusSafety/components/Header';
+
+const ReportFound = () => {
+  const navigate = useNavigate();
+
+  return (
+    <>
+      <Header />
+      <Card>
```
Comment (erandersen21) [inline src/views/CampusSafety/views/LostAndFound/views/ReportFound/index.tsx:12]: This card should probably be centered and sized relative to the content, but I assume you're working on that.

**Review by erandersen21 — APPROVED** [review]
LGTM except for the minor issues I mentioned.

File: src/views/CampusSafety/views/LostAndFound/index.tsx (hunk 1 above)
Comment (Centerville1) [reply src/views/CampusSafety/views/LostAndFound/index.tsx:178]: I just pushed a fix to UI #2442 which fixes this :+1:

File: src/views/CampusSafety/views/LostAndFound/views/ReportFound/index.tsx (hunk 2 above)
Comment (Centerville1) [reply src/views/CampusSafety/views/LostAndFound/views/ReportFound/index.tsx:16]: Yeah, if you have ideas for how to design this page better, let me know.  I kinda agree, but I also don't really know what would work better.  I wrote this page in about 15 minutes, so it's kinda just thrown together.

File: src/views/CampusSafety/views/LostAndFound/views/ReportFound/index.tsx (hunk 3 above)
Comment (Centerville1) [reply src/views/CampusSafety/views/LostAndFound/views/ReportFound/index.tsx:12]: Fixed, good point!

File: src/views/CampusSafety/views/LostAndFound/views/ReportFound/index.tsx (hunk 2 above)
Comment (erandersen21) [reply src/views/CampusSafety/views/LostAndFound/views/ReportFound/index.tsx]: I'm not exactly sure either. Maybe we can all figure out how it should look on Tuesday? I would say you could merge it for now.

**Pull request merged**